index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Porfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="./Images/all.min.css">
+</head>
+<body>
+  <header class="container">
+    <a href="#" class="welcome">My Logo</a>
+    <button type="button"><img class="icon" src="./eclips/menuicon.png" alt="icon"></button>
+  </header>
+  <img class="eclip11" src="./eclips/Ellipse11.png" alt="Eclips">
+  <img class="eclip8" src="./eclips/Ellipse8.png" alt="Eclips">
+  <img class="eclip5" src="./eclips/Ellipse5.png" alt="Eclips">
+  <img class="eclip3" src="./eclips/Ellipse3(1).png" alt="Eclips">
+  <img class="rect4" src="./eclips/Rectangle4.png" alt="small-rectangle">
+  <img class="rect7" src="./eclips/Rectangle7.png" alt="small-rectangle">
+  <section class="section2">
+    <h1>I'm Ahmed <br> Glad to see <br> you!</h1>
+    <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p id="connect">Let's Connect</p>
+    <ul class="socialmeadia-container">
+      <li class="icons"><a href="#"><img src="./svgs/Vector.svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Linkedinicon.svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Vector(4).svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Vector(2).svg" alt="svg"></a></li>
+      <li class="icons"><a href="#"><img src="./svgs/Vector(3).svg" alt="svg"></a></li>
```

Comment:
- [Optional ] I would recommend you format your codes so that it becomes easy for readability reasons.